api/Window.json

```diff
@@ -170,10 +170,10 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": null
+              "version_added": "49"
```

Comment:
The compat table in the page says this is behind the `dom.manifest.onappinstall` pref: https://developer.mozilla.org/en-US/docs/Web/API/Window/onappinstalled#Browser_compatibility.